packages/components/src/custom-select-control/index.js

```diff
@@ -77,6 +79,12 @@ export default function CustomSelectControl( {
 		stateReducer,
 	} );
 
+	const describedByBuild = describedBy
```

Comment:
Nitpick: I'm not sure if we can improve the name of this var.. It's easy to find its usage and understand it, though maybe a name like `controlDescribedBy` or something like that 🤔 would be better?

Reply:
Done.